yarn.lock

```diff
@@ -7065,11 +7065,16 @@ typescript@>=2.8.3:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.5.3.tgz#c830f657f93f1ea846819e929092f5fe5983e977"
   integrity sha512-ACzBtm/PhXBDId6a6sDJfroT2pOWt/oOnk4/dElG5G33ZL776N3Y6/6bKZJBFpd+b05F3Ct9qDjMeJmRWtE2/g==
 
-typescript@^3.2.1, typescript@^3.6.3:
+typescript@^3.2.1:
```

Comment:
The new lock file has two versions of Typescript installed, BTW.

Reply:
aw why would it do that? if 3.9.6 is good enough for both semver ranges, why would 3.9.7 not be?

Reply:
Fixed. It actually had 3 versions :/. Fortunately all this stuff is dev-only.